azext_iot/constants.py

```diff
@@ -54,3 +54,127 @@
 # Initial Track 2 SDK version for DPS
 IOTDPS_MGMT_SDK_PACKAGE_NAME = "azure-mgmt-iothubprovisioningservice"
 IOTDPS_TRACK_2_SDK_MIN_VERSION = "1.0.0"
+
+# Edge device TOML default values
+DEVICE_CONFIG_TOML = {
```

Comment:
I noticed that in another PR we have a new `common.py` file created in `azext_iot/iothub` - do you think this is the right spot for these, or should we:
- Add a `common` folder with (examples):
  - `hub.py`
  - `devices.py`
  - `edge.py`
 - Create a new `edge_common.py` strictly for edge-related common code.
 We could also move some of the `device_identity.py` provider functions into this file , like `_process_nested_edge_config_content` or `create_edge_device_config`

Reply:
What is different about these newly introduced elements is they are focused around device side scripting & configuration which IMO does not harmonize with the organization of common modules in iothub, deviceupdate, digitaltwins, central etc -- those are in the realm of enums (often times to align service & client options), resource ids, naming functions, generic messages (arguably can be moved) and namespace specific client settings.

More prefer the second option, but for these specific attributes, something like `edge_device_config.py` seems appropriate because just reading the module name I have an idea it will be edge device configuration centric where `edge_common` can be ambiguous with respect to what part of edge overall (service side?) + what I was saying earlier where it wouldn't be organized like our other namespace specific common modules.  I also do like the thought of moving the edge config specific functions/classes into an edge config based module rather than mixing with device_identity.